scripts/quests/windurst/To_Catch_A_Falling_Star.lua

```diff
@@ -0,0 +1,154 @@
+-----------------------------------
+-- To Catch A Falling Star
+-----------------------------------
+-- !addquest 2 53
+-- Sigismund : !pos -110 -9 80 206
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require("scripts/globals/items")
+require("scripts/globals/npc_util")
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+
+local westSaruMessages = require("scripts/zones/West_Sarutabaruta/IDs").text
```

Comment:
```suggestion
local westSaruMessages = require("scripts/zones/West_Sarutabaruta/IDs")
```

This .text here is unconventional to code standards. it should be westSaruMessages.text.TEXT_NAME below

Reply:
Will make the change but I suggest updating code standards to shift to the require().something method where the require is only used for a single property in the script.  Enforcing the .text.MESSAGE to be used in every call to a require like that doubles the number of property scans lua must do to find the thing you're looking for.

In relationship to many of the other changes made to match the existing code style/standards, we've essentially double/tripled the number of Heap searches and property scans needed.